srcts/package.json

```diff
@@ -46,6 +46,7 @@
     "setup_build_shiny": "yarn run lint && yarn run typescript-check",
     "build_shiny": "yarn run setup_build_shiny && yarn run bundle_shiny",
```

Comment:
And then the `tools` script could stay with using `yarn build`?


```suggestion
    "build_shiny": "yarn run setup_build_shiny && yarn run bundle_shiny && yarn bundle_external_libs",
```

Reply:
I added it into `yarn build`, rather than `yarn build_shiny`